File: Actividades en el lenguaje Python/Tarea 1 Arreglos-Declarar-Recorrer-Etc.py
#ACTIVIDAD 1: Como declarar un arreglo/inicializarlo, Asignar/modificar valores, Recorrer un arreglo
#Alumno: Cristian Pabel Vidaña Herrera
#Fecha: 27/agosto/2025
#Materia: Estructura de Datos



#Declarar/inicializar un arreglo

Arreglo1 = [0] * 5#Arreglo vacio
Arreglo2 = [1,2,3,4,5] #Arreglo con valores enteros
Arreglo3 = ['a','b','c','d','e'] #Arreglo con valores tipo string
Arreglo4 = [1,'b',3.5,True] #Arreglo con valores mixtos



#Asignar/modificar valores en un arreglo
Arreglo1[0] = 10 



#Recuperacion/leer valores de un arreglo
valor = Arreglo2[2] #Recuperar el valor en la posicion 2 del arreglo2
print("El valor en la posicion 2 del Arreglo2 es:", valor)



#Analisis/Recorrer un arreglo

print("Recorriendo el Arreglo3:")
for elemento in Arreglo3:
    print(elemento)
    

File: Actividades en el lenguaje JavaScript/Tarea_1_Arreglos_Declarar_Recorrer_Etc.js
//ACTIVIDAD 1: Como declarar un arreglo/inicializarlo, Asignar/modificar valores, Recorrer un arreglo
//Alumno: Cristian Pabel Vidaña Herrera
//Fecha: 27/agosto/2025
//Materia: Estructura de Datos

// 1️⃣ Declarar e inicializar un arreglo
let Arreglo1 = [0, 0, 0, 0, 0];  // arreglo de 5 elementos inicializados en 0


// 2️⃣ Asignar o modificar valores
Arreglo1[0] = 10;   // modificar el primer elemento
Arreglo1[1] = 20;   // modificar el segundo elemento
Arreglo1[2] = 30;   // modificar el tercer elemento


// 3️⃣ Recorrer el arreglo e imprimir los valores
console.log("Recorriendo Arreglo1:");
for (let i = 0; i < Arreglo1.length; i++) {
    console.log(Arreglo1[i]);
}


// 🔹 Alternativa usando for-each
console.log("Recorriendo Arreglo1 con forEach:");
Arreglo1.forEach((elemento) => {
    console.log(elemento);
});

